handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslContext.java

```diff
@@ -201,10 +207,32 @@ public ApplicationProtocolConfig.SelectedListenerFailureBehavior selectedListene
             // ignore
         }
         DH_KEY_LENGTH = dhLen;
+
+        CLIENT_DEFAULT_PROTOCOLS = protocols("jdk.tls.client.protocols");
+        SERVER_DEFAULT_PROTOCOLS = protocols("jdk.tls.server.protocols");
+    }
+
+    private static String[] protocols(String property) {
+        String protocolsString = SystemPropertyUtil.get(property, null);
+        if (protocolsString != null) {
+            Set<String> protocols = new HashSet<String>();
+            for (String proto : protocolsString.split(",")) {
+                String p = proto.trim();
+                if (OpenSsl.SUPPORTED_PROTOCOLS_SET.contains(p)) {
```

Comment:
We should log this when the protocol is defined in the `jdk.tls.server.protocols` but not supported by OpenSSL.

Reply:
The JDK is not logging as well... So I think we should also not do it.